feature/sflow/otg_tests/sflow_base_test/README.md

```diff
@@ -0,0 +1,92 @@
+# SFLOW-1: sFlow Configuration and Sampling
+
+## Summary
+
+Verify configuration of sflow and sFlow sample data.
+
+## Procedure
+
+* SFLOW-1.1 Configure sFlow on DUT
+  * Configure DUT and ATE with 2 ports
+  * Configure DUT to send sflow samples to ATE port 2
+  * Set sample source address, sample size 256Bytes, one sample per 1M packets and DSCP=32
+
+* SFLOW-1.2 Send traffic via OTG and verify sFlow packet on OTG
+  * Configure ATE to generate ipv4 and ipv6 traffic and capture sFlow packets
+    * Traffic Profile
+        | Traffic Item | PPS    | Packet Size | L3  | L4  |
+        | ------------ | ------ | ----------- | --- | --- |
+        | sflow3       | 100000 | 64          | IP  | TCP |
+        | mflow3       | 100000 | 512         | IP  | TCP |
+        | lflow3       | 100000 | 1500        | IP  | TCP |
+
+* Verify captured packets are formatted like an sFlow packet
+  * Verify sample size is 256B
+  * Verify 1 sample sent to collector address per 1M packets generated by ATE
+  * Verify sample packet is set with DSCP=32
+
+* SFLOW-1.3 [TODO https://github.com/openconfig/featureprofiles/issues/2346] Additional sflow packet verifications
+  * Using the same packets captured in SFLOW-1.2 verify
```

Comment:
* "Extended-router" container exists and contains accurate details on the following for different flow types, Naked IP v/s IP-IP encaped flows.
          * Next hop
          * Next hop source mask
          * Next hop destination mask

> 

>